lib/ansible/modules/database/aerospike/aerospike_migrations.py

```diff
@@ -0,0 +1,562 @@
+description:
+    - This can be used to check for migrations in a cluster.
+    - This makes it easy to do a rolling upgrade/update on Aerospike nodes.
+    - If waiting for migrations is not desired, simply just poll until
+    - port 3000 if available or asinfo -v status returns ok
+version_added: 2.8
+author: "Albert Autin (@Alb0t)"
+options:
+    host:
+        description:
+            - Which host do we use as seed for info connection
+        required: False
+        type: str
+        default: localhost
+    port:
+        description:
+            - Which port to connect to Aerospike on (service port)
+        required: False
+        type: int
+        default: 3000
+    connect_timeout:
+        description:
+            - How long to try to connect to node before giving up (ms)
+        required: False
+        type: int
+        default: 1000
+    consecutive_good_checks:
+        description:
+            - How many times should the cluster report "no migrations"
+            - consecutively before returning OK back to ansible?
+        required: False
+        type: int
+        default: 3
+    sleep_between_checks:
+        description:
+            - How long to sleep between each check (seconds).
+        required: False
+        type: int
+        default: 60
+    tries_limit:
+        description:
+            - How many times do we poll before giving up and failing?
+        default: 300
+        required: False
+        type: int
+    local_only:
+        description:
+            - Do you wish to only check for migrations on the local node
+            - before returning, or do you want all nodes in the cluster
+            - to finish before returning?
+        required: True
+        type: bool
+    min_cluster_size:
+        description:
+            - Check will return bad until cluster size is met
+            - or until tries is exhausted
+        required: False
+        type: int
+        default: 1
+    fail_on_cluster_change:
+        description:
+            - Fail if the cluster key changes
+            - if something else is changing the cluster, we may want to fail
+        required: False
+        type: bool
+        default: True
+    migrate_tx_key:
+        description:
+            - The metric key used to determine if we have tx migrations
+            - remaining. Changeable due to backwards compatibility.
+        required: False
+        type: str
+        default: migrate_tx_partitions_remaining
+    migrate_rx_key:
+        description:
+            - The metric key used to determine if we have rx migrations
+            - remaining. Changeable due to backwards compatibility.
+        required: False
+        type: str
+        default: migrate_rx_partitions_remaining
+    target_cluster_size:
+        description:
+            - When all aerospike builds in the cluster are greater than
+            - version 4.3, then the 'cluster-stable' info command will be used.
```

Comment:
```suggestion
            - version 4.3, then the C(cluster-stable) info command will be used.
```
(see [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation) for some more formatting options)

Reply:
What is this notation called? never seen this
```C(cluster-stable)

Reply:
Check the link I used above; it contains all formatting directives for module docs: https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation